cosmos/profiles/vertica/user_pass.py

```diff
@@ -1,4 +1,14 @@
-"Maps Airflow Vertica connections using username + password authentication to dbt profiles."
+"""
+Maps Airflow Vertica connections using username + password authentication to dbt profiles.
+
+NOTE: Use Airflow connection `schema` for vertica `database` to keep it consistent with other connection types and profiles.
+Also Vertica Airflow provider hook assumes this:
+https://github.com/apache/airflow/blob/395ac463494dba1478a05a32900218988495889c/airflow/providers/vertica/hooks/vertica.py#L72
+This seems to be a common approach also for Postgres, Redshift and Exasol since there is no `database` field in Airflow connection
+and `schema` is not required for the database connection.
+Posgres Hook:
+https://github.com/apache/airflow/blob/0953e0f844fa5db81c2b461ec2433de1935260b3/airflow/providers/postgres/hooks/postgres.py#L138
```

Comment:
I'd suggest moving this to the `VerticaUserPasswordProfileMapping` docstring with a sphinx note block so it looks better in the Cosmos docs, see suggestion below. 
```python
class VerticaUserPasswordProfileMapping(BaseProfileMapping):
    """
    Maps Airflow Vertica connections using username + password authentication to dbt profiles.
    .. note::
       Use Airflow connection ``schema`` for vertica ``database`` to keep it consistent with other connection types and profiles. \
       The Vertica Airflow provider hook `assumes this <https://github.com/apache/airflow/blob/395ac463494dba1478a05a32900218988495889c/airflow/providers/vertica/hooks/vertica.py#L72>`_.
       This seems to be a common approach also for `Postgres <https://github.com/apache/airflow/blob/0953e0f844fa5db81c2b461ec2433de1935260b3/airflow/providers/postgres/hooks/postgres.py#L138>`_, \
       Redshift and Exasol since there is no ``database`` field in Airflow connection and ``schema`` is not required for the database connection.
    .. seealso::
       https://docs.getdbt.com/reference/warehouse-setups/vertica-setup
       https://airflow.apache.org/docs/apache-airflow-providers-vertica/stable/connections/vertica.html
    """
```
![image](https://github.com/astronomer/astronomer-cosmos/assets/79104794/4c3cab96-10f0-4164-849f-affc311490dc)

If you want to test the cosmos docs locally see the guide [here](https://astronomer.github.io/astronomer-cosmos/contributing.html#writing-docs).